### PR TITLE
Cleans up dashboard, fixes Safari bugs and does a few more cleanups

### LINKED ISSUE #557: The dropdown for action buttons in plio list items needs to be shifted up for plios towards the end of the table

### LINKED ISSUE #558: Dashboard cleanups
- extra white space at the bottom of the page
- add space between video link and buttons
- trim video name to one line with ....
- replace plio link with play plio button
- use play plio and edit button in the same line

### LINKED ISSUE #559: add option text not showing up for safari

### PR DESCRIPTION
**Pull request by dalmia**
Fixes #559 
Fixes #558 
Fixes #557  

## Summary
Look at the issues for details.

## Test Plan

- [x] Test Responsiveness
   - [ ] Laptop (1200px)
   - [ ] Tablet (760px)
   - [ ] Phone (320px)
- [x] Cross-Browser Testing
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Safari (use BrowserStack)
- [x] Local Language Support
- [x] Hygiene and Housekeeping
   - [ ] Self-review
   - [ ] Comments have been added appropriately
   - [ ] Check for bundle size [here](https://bundlephobia.com/) if adding a package
   - [ ] Added relevant details like Labels/Projects/Milestones etc.
- [ ] Testing
   - [ ] Wrote tests
   - [ ] Tested locally
   - [x] Tested on staging
   - [ ] Tested on an actual physical phone
   - [ ] Tested on production
- [x] Lighthouse Checks
   - [ ] Images have `alt` attributes
   - [ ] Any `` tags have `width` and `height` specified
   - [ ] Any `target="_blank"` links have `rel="noopener"`
   - [ ] Only SVGs are used as images. If PNGs are used, their size has been optimised.
   - [ ] Any SVGs without text have their `aria-label` attributes set

### CODE REVIEW

File: tests/unit/pages/Dashboard.spec.js [hunk 1]
```diff
@@ -156,77 +69,113 @@ describe("Dashboard.vue", () => {
...
+          global.dummyPlioAnalytics[
+            "GroupedSessionRetention.averageOneMinuteRetention"
+          ]
+        ) + "%"
+      );
+    });
 
-    await wrapper.find('[data-test="edit"]').trigger("click");
+    it("renders analytics values when none available", async () => {
+      await mountWrapper({});
```
Comment (deepansh96) [inline tests/unit/pages/Dashboard.spec.js:135]: no unmount before mounting?

File: tests/unit/pages/Dashboard.spec.js [hunk 2]
```diff
@@ -156,77 +69,113 @@ describe("Dashboard.vue", () => {
...
+            org: "",
+            plioId: plioId,
+          },
+        });
+      });
 
-    await wrapper.find('[data-test="download"]').trigger("click");
+      it("downloads report", async () => {
+        const downloadReport = jest.spyOn(Dashboard.methods, "downloadReport");
+        await mountWrapper({});
```
Comment (deepansh96) [inline tests/unit/pages/Dashboard.spec.js:173]: no unmount before mounting?

File: src/components/Collections/ListItems/PlioListItem.vue [hunk 3]
```diff
@@ -309,10 +305,8 @@ export default {
      * 3. touch screen device with screen width >= 640
      */
     setOptionsOverflowMarginTop() {
-      if (this.isTouchDevice) {
-        if (this.isTabScreen) this.optionsOverflowMarginTop = -18;
-        else this.optionsOverflowMarginTop = -16;
-      } else this.optionsOverflowMarginTop = -14;
```
[the remark this answers is not in the document]
Comment (dalmia) [reply src/components/Collections/ListItems/PlioListItem.vue:315]: comment here

**Review by deepansh96 — APPROVED** [review]

File: src/pages/Dashboard.vue [hunk 4]
```diff
@@ -368,15 +368,22 @@ export default {
         return Math.round(this.plioAnalytics["num-questions-answered"]);
       return "0";
     },
+    playButtonTextConfig() {
+      return {
+        value: this.$t("dashboard.buttons.play"),
+        class: "text-sm md:text-md lg:text-lg text-white p-2",
+      };
+    },
+    playButtonClass() {
```
Comment (dalmia) [inline src/pages/Dashboard.vue:377]: why computed?

**Pull request merged**